packages/components/src/local/organisms/setting-forces/__snapshots__/index.spec.tsx.snap

```diff
@@ -339,7 +298,7 @@ exports[`SettingForces component: renders correctly 1`] = `
                                     >
                                       <span
                                         aria-disabled={false}
-                                        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-17 MuiSwitch-switchBase WithStyles(ForwardRef(Switch))-switchBase-14 MuiSwitch-colorSecondary PrivateSwitchBase-checked-18 Mui-checked WithStyles(ForwardRef(Switch))-checked-15"
```

Comment:
@lebaphi  - do you know why all of these index numbers have changed? 

I _think_ there's something we can do to avoid the problem.